socorro/processor/processor_pipeline.py

```diff
@@ -91,7 +91,7 @@ class ProcessorPipeline(RequiredConfig):
     )
     required_config.breakpad.add_option(
         "kill_timeout",
-        doc="amount of time to let mdsw run before declaring it hung",
+        doc="amount of time in seconds to let mdsw run before declaring it hung",
```

Comment:
While I was fixing this bug, I figured I'd clarify that the value is in seconds here so future me doesn't have to look it up again.